setuptools/tests/test_wheel.py

```diff
@@ -547,7 +553,7 @@ def __repr__(self):
 @pytest.mark.parametrize(
     'params',
     WHEEL_INSTALL_TESTS,
-    ids=list(params['id'] for params in WHEEL_INSTALL_TESTS),
+    ids=[params['id'] for params in WHEEL_INSTALL_TESTS],
```

Comment:
Not a necessary change, but I noticed this improvement whilst I was looking at this line. Would've been caught by https://docs.astral.sh/ruff/rules/unnecessary-generator-list/

Reply:
I'm good with it. Thanks for highlighting!